mne/cov.py

```diff
@@ -51,8 +51,7 @@ def _check_covs_algebra(cov1, cov2):
 
 def _get_tslice(epochs, tmin, tmax):
     """get the slice."""
-    tstart, tend = None, None
-    mask = _time_mask(epochs.times, tmin, tmax, sfreq=epochs.info['sfreq'])
+    mask = _time_mask(epochs.times, tmin, tmax, strict=True)
```

Comment:
Yeah, strict was only used on some operation on ``SourceEstimate``. Didn't dare change it. Will check monday..